certifier-osv.md

```diff
@@ -0,0 +1,207 @@
+```bash
+guacone certifier osv --sbom <path> [options]
+```
+
+### Flags
+```bash
+Flags:
+      --certifier-batch-size int   sets the batch size for pagination query for the certifier (default 60000)
+      --certifier-latency string   sets artificial latency on the certifier. Defaults to empty string (not enabled) but can set m, h, s...etc
+  -h, --help                       help for osv
+  -l, --last-scan int              hours since the last scan was run. If not set, run on all packages/sources (default 4)
+```
+### Global Flags:
+```bash
+      --add-license-on-ingest   if enabled, the ingestor will query and ingest clearly defined for licenses. Warning: This will increase ingestion times
+      --add-vuln-on-ingest      if enabled, the ingestor will query and ingest OSV for vulnerabilities. Warning: This will increase ingestion times
+      --csub-addr string        address to connect to collect-sub service (default "localhost:2782")
+      --csub-tls                enable tls connection to the server
+      --csub-tls-skip-verify    skip verifying server certificate (for self-signed certificates for example)
+      --gql-addr string         endpoint used to connect to graphQL server (default "http://localhost:8080/query")
+      --header-file string      a text file containing HTTP headers to send to the GQL server, in RFC 822 format
+  -i, --interval string         if polling set interval, m, h, s, etc. (default "5m")
+  -p, --poll                    sets the collector or certifier to polling mode
+```
+
+## Output Format
+
+### Vulnerability Report Fields
+
+| Field | Description | Example |
+|-------|-------------|---------|
+| id | OSV vulnerability identifier | `OSV-2023-001` |
+| package | Affected package name | `example-library` |
+| version | Affected version | `1.2.3` |
+| severity | Vulnerability severity | `High` |
+| remediation | Fix instructions | `Update to version 1.2.4 or later` |
+
+### Sample Output
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [
+    {
+      "uri": "pkg:npm/example-library@1.2.3"
+    }
+  ],
+  "predicateType": "https://in-toto.io/attestation/vulns/v0.1",
+  "predicate": {
+    "scanner": {
+      "uri": "osv.dev",
+      "version": "0.0.14",
+      "result": [
+        {
+          "id": "GHSA-rc38-5r82-hr3j"
+        },
+        {
+          "id": "CVE-2023-12345"
+        }
+      ]
+    },
+    "metadata": {
+      "scanStartedOn": "2023-06-06T06:15:28Z",
+      "scanFinishedOn": "2023-06-06T06:15:28Z"
+    }
+  }
+}
+```
+
+### Remediation Process
+
+1. **Prioritize**: 
+   - Review vulnerability severity
+   - Assess impact on application
+   - Consider dependency update complexity
+
+2. **Update**: 
+   - Follow remediation guidance
+   - Test compatibility of new versions
+   - Update dependency declarations
+
+3. **Verify**: 
+   - Rescan with OSV Certifier
+   - Confirm vulnerability resolution
+   - Update SBOM documentation
+
+## Error Handling
```

Comment:
I like idea of including possible errors, but the content here is too generic to be valuable. It might be worth dropping it for now.

Reply:
Yeah sure, dropped the Error Handling part